README.md

```diff
@@ -112,7 +112,7 @@ window.injectedWeb3 = {
   // each with their own keys, here `aleph-zero-signer` is for this extension
   'aleph-zero-signer': {
     // semver for the package
-    version: '1.0.0',
+    version: '{{signer-version}}',
```

Comment:
seems like and overkill to me :stuck_out_tongue:

Reply:
Ok, I can revert :D Was wondering about it for some time 😆